src/custom-element.js

```diff
@@ -1156,6 +1169,7 @@ function createBaseCustomElementClass(win) {
       this.preconnect(/* onLayout */true);
       this.classList.add('i-amphtml-layout');
       return promise.then(() => {
+        blankBoxCounter.layoutComplete(this.getResource());
```

Comment:
I think this code should be moved to `Resources` class. Also, I think we should just sample it at 1% to start with.